content/docs/sidebar.json

```
@@ -154,6 +154,9 @@
           "share-a-dvc-cache"
         ]
       },
+      {
+        "slug": "visualizing-plots"
+      },
```

This can be shortened to just

```suggestion
      "visualizing-plots"
```

p.s. there's an unnecessary new line in line 167.

---

UPDATE: I'm fixing these in https://github.com/iterative/dvc.org/pull/3903

---

> I already submitted a PR in https://github.com/iterative/dvc.org/pull/3902

Oops. My bad